stl/src/vector_algorithms.cpp

```diff
@@ -2075,6 +2075,75 @@ namespace {
         }
         return _Result;
     }
+
+    template <class _Ty>
+    const void* __stdcall __std_find_first_of_trivial_impl(
+        const void* _First1, const void* const _Last1, const void* const _First2, const void* const _Last2) noexcept {
+#ifndef _M_ARM64EC
+        constexpr bool _Bytes = sizeof(_Ty) == 1;

```

Comment:
Suggestion: `_Bytes` sounds like a number of bytes. We recently used `_Single_byte_elements` which would be consistently descriptive here.

Reply:
It is only used twice in other `constexpr` below, so we can skip naming it